src/Facility.CodeGen.Python/CodeTemplateUtility.cs

```diff
@@ -19,8 +19,7 @@ public static string Render(string templateText, CodeTemplateGlobals globals)
 			var text = template.Render(templateContext);
 
 			text = Regex.Replace(text, @"[ \t]+\n", "\n");
-			text = Regex.Replace(text, @"\n\n+", "\n\n");
-			text = Regex.Replace(text, @"\n+$", "\n");
```

Comment:
Was this line removed intentionally? If so, why?

Reply:
White space is significant in Python. I needed to allow double blank lines. However, I didn't need to remove line 23 if it only removes blank lines at the end of the file.